stdlib/ysh/args.ysh

```diff
@@ -80,18 +80,17 @@ proc parser (; place ; ; block_def) {
   call place->setValue(p)
 }
 
-const kValidTypes = [Bool, Float, Int, Str]
+const kValidTypes = [Bool, Float, List[Float], Int, List[Int], Str, List[Str]]
 const kValidTypeNames = []
 for vt in (kValidTypes) {
-  call kValidTypeNames->append(vt.name)
+  var name = vt.name if ('name' in propView(vt)) else vt.unique_id
```

Comment:
ok this is interesting ... I was wondering if they should be the same field name, but I sorta wanted to distinguish between "primitive types" and "parameterized types that are computed from primitive types"

But I guess this is only used for printing strings, so it might be OK

----

I also wonder if we should get rid of the `propView()` everywhere, by treating Obj more like Dict

BTW I also have this idea of turning `prototype()` and `propView()` into `first() rest()`.  I think those names kinda make more sense

I took the "prototype" name from JS, but a linked list naming convention  might be better